pace-util/pace/util/__init__.py

```diff
@@ -44,6 +44,7 @@
 from .halo_updater import HaloUpdater
 from .initialization import GridSizer, QuantityFactory, SubtileGridSizer
 from .io import read_state, write_state
+from .local_communicator import LocalComm
```

Comment:
Can you rename the file to `local_comm.py`, for consistency?

Reply:
Sure